server/lib/config/pbench-server-default.cfg

```diff
@@ -25,6 +25,12 @@ admin-email=%(user)s@localhost
 mailto=%(admin-email)s
 mailfrom=%(user)s@localhost
 
+# A list of usernames to be given ADMIN access on the server. These are SSO ID
+# provider usernames matched against decrypted OIDC2 authorization tokens. If
+# not specified, no users have ADMIN access. NOTE: this is a temporary measure
+# until we work out Keycloak / LDAP roles.
+#admin-role=user1,user2
```

Comment:
We should probably either mention that the list must contain no whitespace or we should have the code strip any that it finds.

Reply:
I didn't want to get fancy on parsing, since this isn't "user input".

Reply:
I don't mind the lack of fancy parsing, so long as nobody is surprised by it.  (The updated comment covers it.)

Whether this is "user input" or not depends on who you think the "user" is here.  (I reserve great pity for the "admin" who has to fill in this field, because, at some point, it'll probably be me!)